src/pages/Browse/index.spec.tsx

```diff
@@ -336,11 +341,11 @@ describe(__filename, () => {
     expect(dispatchSpy).toHaveBeenCalledTimes(1);
   });
 
-  it('does not dispatch anything on update when nothing has changed', () => {
-    const { renderAndUpdate } = setUpVersionFileUpdate();
+  it('does not dispatch fetchVersion on update when nothing has changed', () => {
+    const { fakeThunk, renderAndUpdate } = setUpVersionFileUpdate();
     const { dispatchSpy } = renderAndUpdate();
 
-    expect(dispatchSpy).not.toHaveBeenCalled();
+    expect(dispatchSpy).not.toHaveBeenCalledWith(fakeThunk.thunk);
```

Comment:
I'm surprised that these changes needed to be made to this file considering that there were no changes made to the `Browse` page in this patch. Is this fixing a problem that already existed in this test file?

Reply:
Yeah, it improves the test slightly. I actually needed it when an earlier version of the patch was dispatching more actions in the same constructor. This change isn't necessary now but it will make future expansion of the action dispatching easier so I think it's worth keeping.